### PR TITLE
Docs review cont.; Post release change log update

### PR DESCRIPTION
**Pull request by uellue**
Still reviewing, but can be merged any time :-)

## Contributor Checklist:

* [x] I have added or updated my entry in [the creators.json file](https://github.com/LiberTEM/LiberTEM/blob/master/packaging/creators.json)
* [x] I have added [a changelog entry](https://github.com/LiberTEM/LiberTEM/tree/master/docs/source/changelog) for my contribution
* [x] I have added/updated documentation for all user-facing changes
* [x] I have added/updated test cases

### CODE REVIEW

File: docs/source/api.rst [hunk 1]
```diff
@@ -35,9 +35,8 @@ The :meth:`~libertem.contrib.daskadapter.make_dask_array` function can generate
 .. testsetup:: *
 
     from libertem import api
-    from libertem.executor.inline import InlineJobExecutor
 
-    ctx = api.Context(executor=InlineJobExecutor())
+    ctx = api.Context()
```
Comment (sk1p) [inline docs/source/api.rst:39]: Apparently this triggers an "unexpected EOF" error?

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -23,8 +23,11 @@ Changelog
 .. _latest:
 .. _`v0-5-0`:
 
-0.5.0.rcX / 2020-4-WIP
-######################
+0.5.0 / 2020-4-23
```
Comment (sk1p) [inline docs/source/changelog.rst:26]: Nitpick: would prefer full ISO format:
```suggestion
0.5.0 / 2020-04-23
```

File: docs/source/api.rst (hunk 1 above)
Comment (uellue) [reply docs/source/api.rst]: I'm trying to use a Dask client that doesn't use multiprocessing. Let's see what CI says!

**Comment by uellue** [comment]
@sk1p I'm boldly going ahead and merging this so that we can finish the release. :-)

**Pull request merged**